iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/receiver/protocol/pipeconsensus/PipeConsensusReceiver.java

```diff
@@ -1076,35 +1124,54 @@ private class RequestExecutor {
     // An ordered set that buffers transfer requests' TCommitId, whose length is not larger than
     // PIPE_CONSENSUS_PIPELINE_SIZE.
     // Here we use set is to avoid duplicate events being received in some special cases
-    private final TreeSet<TCommitId> reqExecutionOrderBuffer;
+    private final TreeSet<RequestMeta> reqExecutionOrderBuffer;
     private final Lock lock;
     private final Condition condition;
-    private long onSyncedCommitIndex = -1;
+    private final PipeConsensusReceiverMetrics metric;
+    private long onSyncedCommitIndex = 0;
     private int connectorRebootTimes = 0;
+    private int WALEventCount = 0;
+    private int tsFileEventCount = 0;
```

Comment:
Better use volatile here because these two are got by the metrics framework.

Reply:
ok